validate.go

```diff
@@ -71,3 +71,34 @@ func ComposeValidators(validators ...Validator) Validator {
 		return nil
 	}
 }
+
+// isZero returns true if the passed value is the zero object
+func isZero(v reflect.Value) bool {
```

Comment:
So I replaced the function with the following body:

```golang

// Required does not allow an empty value
func Required(val interface{}) error {
	// reflect the value
	v := reflect.ValueOf(val)
	// if the value passed in is the zero value of the appropriate type
	if reflect.DeepEqual(v.Interface(), reflect.Zero(v.Type()).Interface()) {
		return errors.New("Value is required")
	}

	return nil
}
```

And now the list test is failing, this looks a lot like what I had before, but I think this won't work for the same reason - our `MultiChoice` prompt returns an empty list of  answers, not `nil` which is what I think the `Zero()` value is of an array.